README.md

```diff
@@ -174,11 +174,18 @@ That nodeSelector is a top level configuration option affecting all the active d
 this profile is not expected to be combined with other profiles
 unless all profiles are expected to operate over the same set of nodes.
 
+This profiles, by default, deploys the soft-tainter component to dynamically set/remove soft taints according to the
+same criteria used for load aware descheduling. In case of load-aware descheduling we can potentially have a
+relevant asymmetry between the descheduling and successive scheduling decisions.
+The soft taints set by the descheduler soft-tainter act as a hint for the scheduler to mitigate
+this asymmetry and foster a quicker convergence.
+The SoftTainter component can be eventually disabled with the KubevirtRelieveAndMigrateSoftTainter Profile Customizations.
+
 The profile exposes the following customization:
 - `devLowNodeUtilizationThresholds`: Sets experimental thresholds for the LowNodeUtilization strategy.
 - `devActualUtilizationProfile`: Enable load-aware descheduling.
 - `devDeviationThresholds`: Have the thresholds be based on the average utilization.
-- `devEnableSoftTainter`: Have the operator deploying the soft-tainter component to dynamically set/remove soft taints according to the same criteria used for load aware descheduling. Scheduling and descheduling decisions are in general fully independent. In case of load-aware descheduling we can potentially have a relevant asymmetry between the descheduling and successive scheduling decisions. The soft taints set by the descheduler soft-tainter act as a hint for the scheduler to mitigate this asymmetry and foster a quicker convergence.
+- `KubevirtRelieveAndMigrateSoftTainter`: Control if the profile will trigger the deployment of the soft-tainter component. Enabled by default.
```

Comment:
All the other customizations are prefixed with `dev`, to drop the prefix for this one is by design ?

Reply:
That is OK. I think you have a typo here, it should be `kubevirtRelieveAndMigrateSoftTainter` with a lower case.

Reply:
done